TACC/Core-Portal
# task/FP-1294: Toggle Site Search Systems

## Overview: ##
Adds the property `'siteSearchPriority'` to systems in `_PORTAL_DATAFILES_STORAGE_SYSTEMS`. When `'siteSearchPriority'` is not present, that system will not be included in Site Search.

## Related Jira tickets: ##

* [FP-1294](https://jira.tacc.utexas.edu/browse/FP-1294)

## Summary of Changes: ##
- [catch missing access to tapis system](https://github.com/TACC/Core-Portal/pull/571/commits/da1dfcf5b273de02be934da18e147859ccbd8326)
- Redirect invalid site search urls
- add `siteSearchPriority` property to datafiles systems

## Testing Steps: ##
1. Remove `'siteSearchPriority'` on the Public Data system in `_PORTAL_DATAFILES_STORAGE_SYSTEMS` in `settings_default.py`
2. Go to site search: https://cep.dev/search/cms/?page=1&query_string=test
3. Confirm that Public Data does not show up

## UI Photos:

## Notes: ##
